### ISSUE TITLE
saveing as temp files instead of the actual files 

### ISSUE THREAD

**Issue by flarechu1** [Bionus/imgbrd-grabber#2082]
**Bug description**
trying to mass download from a few sites, may be files i have downloaded befor.. desided to deleat what i had and redownload with more tags  and now its all downloading as temp files.
i manually downloaded a pic from grabber, it saved as tmp and i just renamed the file taking away the .tmp and it was the full file. very confused.
[main.log](https://github.com/Bionus/imgbrd-grabber/files/5075272/main.log)

**Steps to reproduce**

**Context**
[main.log](https://github.com/Bionus/imgbrd-grabber/files/5075272/main.log)

- OS: Windows 10
- Grabber version: 7.3.2

**Comment by zura64** [comment]
i have the same problem, i posted here almost a month ago and still no solution (issue 2049)

**Comment by omegarunner1** [comment]
What sites did you attempt to download from? Because I only attempt to download from rule34.xxx and I also run the program on Windows 7 and thus far I've never run into such a glitch. I do know based both on my own experiences and the experiences of others that rule34.paheal.net and pixiv are seemingly plagued with issues perpetually but that's the extent of my knowledge.

**Comment by zura64** [comment]
From danbooru and yandere. I uninstalled and reinstalled and used the default settings and i still have the problem. Something on my side must be causing it, maybe an update or something, no idea how to check.

**Comment by omegarunner1** [comment]
What the error log is telling me is that the downloads aren't finishing. I don't know about other browsers or Operating Systems but I know that when I start downloading images and videos, the very first thing to download will be a .tmp file, which will then be removed upon finishing the download. So there may be something going on in terms of your internet connection. In addition, I have been having issues recently with certain images just not showing up in grabber at all. As in, the program finds them, it knows their location but, for whatever reason, Grabber itself simply can't load them and can't download them as a result, that may be tied to this somehow. In other words, it may be a problem with some very specific images. If this is the case, it may be something worth looking in to.

**Comment by flarechu1** [comment]
its not just us, i'v had a few people try as well but have the same issues even with a fresh install

**Comment by omegarunner1** [comment]
I'm sorry about that. Unfortunately, I haven't been running into the issue that you guys have. So all I can offer is speculation.

**Comment by Flibit441** [comment]
I was getting this for a while until I noticed that the issue was only occurring in subfolders of a specific directory on my PC. In my case, any new images would save as *.correctExtension.tmp if they were saved in and subfolder found in "I:\BACK OFF!!!\Pics\Grabber\#3D\*". My default directory is "I:\BACK OFF!!!\Pics\Grabber\" and I sort images by subfolders. Other subfolders in this location were fine, just the #3D one had issues. Any subfolders under #3D would only save the .tmp files. I'm unsure what about that location Grabber didn't like, I thought it might be the short folder name, but other variation on the directory (ex. "I:\BACK OFF!!!\Pics\Grabber\#2D\*" or "I:\BACK OFF!!!\Pics\Grabber1\#3D\*") would work. It was only occurring in folders in that specific location. I've since just renamed the folder to "#3D imgs" and ran a find and replace for the directories in my monitors.json file and no more .tmp files!

Additionally, I just tried making a back up of my options folder, deleted the original, did a fresh install to bring things back to default settings, and I could still reproduce the issue in that install location even on a fresh install.

tl;dr Try changing the save destination folder and this may resolve for you.
